src/consensus/params.h

```diff
@@ -1,5 +1,5 @@
 // Copyright (c) 2009-2010 Satoshi Nakamoto
-// Copyright (c) 2009-2016 The Bitcoin Core developers
+// Copyright (c) 2009-2017 The Bitcoin Core developers
```

Comment:
I have a habit of updating the copyright dates whenever I modify a file.
What do you mean by done separately - should I not update them?